examples/AspNetCore/Program.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-using System.Reflection;
+using Examples.AspNetCore;
```

Comment:
"using" or "namespace"?

Reply:
I think `using` is correct. This allows me to reference `Instrumentation` here. As far as I know you cannot define a namespace with these top level statements